### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by gogasca**
Adding docker support.
Existing docker base image already supports [this](https://github.com/googleapis/testing-infra-docker/pull/10)
Require for Cloud AI repo [ai-platform-samples](https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/training/pytorch/structured/custom_containers/base/scripts/train-local.sh#L38)

### CODE REVIEW

File: python/googleapis/python-multi/Dockerfile [hunk 1]
```diff
@@ -129,4 +129,22 @@ ENV PATH /google-cloud-sdk/bin:$PATH
...
 RUN sudo systemctl enable redis-server.service
 
+# Install docker
+RUN apt-get update \
+  && apt-get install -y \
+     apt-transport-https \
+     ca-certificates \
+     curl \
+     software-properties-common \
+     sudo
```
Comment (kurtisvg) [inline python/googleapis/python-multi/Dockerfile]: These should probably be added to the install set at L28 (a few are already there).

File: python/googleapis/python-multi/Dockerfile [hunk 2]
```diff
@@ -129,4 +129,22 @@ ENV PATH /google-cloud-sdk/bin:$PATH
...
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+
+RUN apt-get update && apt-get install -y docker-ce
```
Comment (kurtisvg) [inline python/googleapis/python-multi/Dockerfile]: I think you can probably move this into the install chain above as well, you'll just need to move the key and add-apt/repository higher up in the dockerfile.

Comment (gogasca) [reply python/googleapis/python-multi/Dockerfile]: To install docker I need `curl` and `sudo` packages, is it ok to move the dependencies to the chain as suggested in previous comment and have a separate section for Docker itself?

```
# Install docker
RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -

RUN add-apt-repository \
   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
   $(lsb_release -cs) \
   stable"

RUN apt-get update && apt-get install -y docker-ce
```

Comment (kurtisvg) [reply python/googleapis/python-multi/Dockerfile]: That seems reasonable to me - you probably should clear the apt-get caches and stuff like the above command does though

Comment (gogasca) [reply python/googleapis/python-multi/Dockerfile]: Thanks, just submitted update.

**Pull request merged**